pkg/catalogdiagrams/diagrams.go

```diff
@@ -77,7 +77,7 @@ func (v *DataModelView) GenerateDataView(dataParam *DataModelParam, appName stri
 			}
 			v.DrawPrimitive(viewParam, pe.String(), relationshipMap)
 		} else if seq := entityType.GetSequence(); seq != nil {
-
+			// TODO
```

Comment:
Nit: elaborate or remove

Reply:
Forget to push the updated demo file.
The original sysl spec is 
```
    !alias FooRequest:
        string
```
so it's supposed to have no fields. But the old implementation is add an `id` as placeholder.